_posts/2020-02-26-PC-Engines-Firmware-v4-11-0-4.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.11.0.4"
+date:   2020-02-26
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.11.0.4
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** e53f8c9.
+2. Fixed **SD card boot order** on apu5
+3. Added **ARM management controller interaction** on apu5
+
+Legacy:
+
+1. Fixed **SD card boot order** on apu5
+2. Added **ARM management controller interaction** on apu5
+
+## coreboot community
+
+Patches sent for review:
+
+* [mb/pcengines/*/devicetree: remove non-existing NCT5104d LDN 0xe](https://review.coreboot.org/c/coreboot/+/38851)
+* [nb/amd/{agesa,pi}/acpi: include thermal zone](https://review.coreboot.org/c/coreboot/+/38755)
+* [superio/nuvoton/nct5104d: add chip config option to reset GPIOs](https://review.coreboot.org/c/coreboot/+/38850)
+
+Patches merged by community:
+
+* [mb/pcengines/apu2: use AGESA 1.0.0.4 with adjusted AGESA header](https://review.coreboot.org/c/coreboot/+/35906)
+* [sb/amd/{agesa,pi}/hudson/Kconfig: Change default SATA mode to AHCI](https://review.coreboot.org/c/coreboot/+/35891)
+* [nb/amd/pi/00730F01: enable ACS and AER for PCIe ports](https://review.coreboot.org/c/coreboot/+/35313)
+
+**Total:**
+
+* 1182 lines added,
```

Comment:
@piotr-kleins this statistic should cover only merged patches. Please update

Reply:
updated